tarodb/sqlc/db.go

```diff
@@ -24,7 +24,7 @@ type Queries struct {
 	db DBTX
 }
 
-func (q *Queries) WithTx(tx *sql.Tx) *Queries {
+func (q *Queries) WithTx(tx DBTX) *Queries {
```

Comment:
Ah ok, I see it's modifying the generated sqlc query interface.

Reply:
Yeah, this is just a very small replace in the generated code that allows us to only reference the `sqlc.DBTX` in the query generators instead of the more specific `*sql.Tx`. But I'm not sure if it's worth doing, doesn't solve anything specific for us right now.

Reply:
> But I'm not sure if it's worth doing, doesn't solve anything specific for us right now.

Yeah I'd rather just drop it, and live with the `*sql.Tx` leak for now. Haven't really run into anything yet that actually gets annoyed by it, so maybe we'd just add this stuff to never really use it later.

Reply:
Okay, dropped this change.